### ISSUE TITLE
newlines giving strange results in our result list

### ISSUE THREAD

**Issue by rduivenvoorde** [geopython/MetaSearch#59]
It seems that certain esri filled metadata is full of newlines, giving strange results in our result list.

For example search for 'Kunst' in the Nationaal Georegister. The first record then is several lines high.

![screenshot from 2014-06-06 10 43 54](https://cloud.githubusercontent.com/assets/731673/3198255/bf69d2e4-ed56-11e3-8008-cb7872f33e45.png)

If you look at the xml in a browser:
http://www.nationaalgeoregister.nl/geonetwork/srv/eng/csw?outputFormat=application%2Fxml&service=CSW&outputSchema=http%3A%2F%2Fwww.opengis.net%2Fcat%2Fcsw%2F2.0.2&request=GetRecordById&version=2.0.2&elementsetname=full&id=5687365c-b343-4820-b0df-02757ba9c85c

you see teh dc:type on one line, but if you look into the source, you see it has a lot of newlines in it:

dc:typeVector

```
    Simple
    Multipoint
    FALSE
    41
    TRUE
    FALSE


    Composite object
    41</dc:type>
```

So, isn't it best to remove newlines from this result items? Or at least from 'type' but I would think also from 'title'?

**Comment by tomkralidis** [comment]
@rduivenvoorde fixed in master (1f396d58ce3f57943d57e019b03d27d5134fb65d).  Let me know if this works for you.

**Comment by rduivenvoorde** [comment]
yep working :-)

closing this one
